ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessAttachmentServiceSpec.scala

```diff
@@ -12,6 +12,7 @@ import pl.touk.nussknacker.ui.config.AttachmentsConfig
 import pl.touk.nussknacker.ui.db.entity.AttachmentEntityData
 import pl.touk.nussknacker.ui.process.repository.{DbProcessActivityRepository, ProcessActivityRepository}
 import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.listener.{Comment => CommentValue}
```

Comment:
This rename is necessary in all places? I thought that the only place where it is needed is in `ProcessActivityRepository` where the same class exists. Please pick one of choices:
1. Use rename only in places where it is needed to avoid confusion of names
2. Rename this class

Reply:
you were right, changed it, now it's only in ProcessActivityRepository